pkg/securitypolicy/framework.rego

```diff
@@ -223,6 +227,12 @@ create_container := {"matches": matches, "env_list": env_list, "started": starte
     ]
 
     count(containers) > 0
+
+    allow_stdio_access := containers[0].allow_stdio_access
```

Comment:
Does this imply that the first container listed in the container array sets the defacto default for all other containers?

Reply:
No. This is narrowing logic. It means that if we can't when narrowing find that all possible containers have the same value then we dont allow the container to start because it is undecidable what item from the policy should be used.

Matt and I are discussing the best way to explain close to this code.